src/Controller/UserController.php

```diff
@@ -178,7 +178,7 @@ public function viewProfileAction(Request $req)
         }
 
         return $this->container->get('templating')->renderResponse(
```

Comment:
should use `$this->render()`, which calls Twig without the `templating` wrapper which is deprecated